### PR TITLE
Privacy Policy updates

### PR DESCRIPTION
**Pull request by tarikbellamine**
### Description
Update privacy disclosures and add relevant links.

### Other changes
N/A

### Tested
Manually.

The one thing I wasn't able to test was the Android contacts list disclosure text because my device did not prompt it.

### How others should test
Test that links work in Settings and ToS

### Related issues
N/A

### Backwards compatibility
Yes

simulator_screenshot_D5CC72C6-4E80-4637-8429-4A64D4FF1E34

simulator_screenshot_821E326E-A7C8-479C-AF77-AF5258213D98

### CODE REVIEW

File: packages/mobile/locales/en-US/nuxNamePin1.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
...
-    "goldDisclaimer": "When you create an ‘account’ with {{appName}} you are creating a digital wallet to which only you hold the keys.  No other person or entity, including cLabs, can recover your key, change or undo transactions, or recover lost funds. Be aware that CELO and cUSD are part of a new asset class and present a risk of financial loss.  Carefully consider your financial circumstances and tolerance for financial risk before purchasing either. You can only send up to {{dailyLimitCUSD}} cUSD per day through {{appName}} (the “Daily Limit”).  You must contact support@cLabs.co and provide additional, personal information to request increasing this Daily Limit. There could be a delay to conduct required compliance."
+    "privacy": "By joining this network, you give us permission to collect anonymous information about your use of the app. Additionally, if you confirm your phone number, a hashed copy of it will be stored on the Celo Network. If you grant Valora access to your contact list, Valora will import contacts name, phone number and profile picture as described in our <0>Privacy Policy</0>.",
```
Comment (silasbw) [inline packages/mobile/locales/en-US/nuxNamePin1.json:13]: I'm tripping on the grammar here.  should this is possessive (`contacts'`)?  do we need an article?  How about:

` ...Valora will import each contact's name ....`

**Review by silasbw — APPROVED** [review]
A w/ q on copy.

File: packages/mobile/locales/en-US/nuxNamePin1.json [hunk 2]
```diff
@@ -10,8 +10,8 @@
...
+    "goldDisclaimer": "When you create an ‘account’ with {{appName}} you are creating a digital wallet to which only you hold the keys.  No other person or entity, including cLabs or Valora, can recover your key, change or undo transactions, or recover lost funds. Be aware that CELO, cUSD, and cEUR are part of a new asset class and present a risk of financial loss.  Carefully consider your financial circumstances and tolerance for financial risk before purchasing any asset. You can only send up to {{dailyLimitCUSD}} cUSD per day through {{appName}} (the “Daily Limit”).  You must contact support@cLabs.co and provide additional, personal information to request increasing this Daily Limit. There could be a delay to conduct required compliance."
```
Comment (MuckT) [inline packages/mobile/locales/en-US/nuxNamePin1.json:14]: `When you create an ‘account’ with {{appName}} you are creating a digital wallet to which only you hold the keys.  No other person or entity, including cLabs or Valora, can recover your key, change or undo transactions, or recover lost funds. Be aware that CELO, cUSD, and cEUR are part of a new asset class and present a risk of financial loss. Carefully consider your financial circumstances and tolerance for financial risk before purchasing any asset. You can only send up to {{dailyLimitCUSD}} cUSD per day through {{appName}} (the “Daily Limit”).  You must contact support@valorapp.com and provide additional, personal information to request increasing this Daily Limit. There could be a delay to conduct required compliance.`

Removed a double space and changed support@clabs.co to support@valoraapp.com

File: packages/mobile/locales/en-US/nuxNamePin1.json (hunk 1 above)
Comment (tarikbellamine) [reply packages/mobile/locales/en-US/nuxNamePin1.json:13]: Yeah I thought that wording was weird...will change

File: packages/mobile/locales/en-US/nuxNamePin1.json (hunk 2 above)
Comment (tarikbellamine) [reply packages/mobile/locales/en-US/nuxNamePin1.json]: Thanks

**Comment by ValoraQA** [comment]
Hi @tarikbellamine, We have verified this task on latest Android internal build V1.19.0 & Testflight build V1.19.0 (1029460203) and updated the status in below sheet;
**Sheet link:** https://docs.google.com/spreadsheets/d/1bWi6smdTMClreG96sGjYxb2lee9Ja6eT4-Lm7eXm1gM/edit#gid=1673025569

**Devices:** iPhone 12 (14.7.1), Google Pixel 2XL (11.0)

**Comment by ValoraQA** [comment]
Hi @gnardini , We have verified this task on latest Android internal build V1.19.0 & updated the status in below sheet.

**Sheet link:** https://docs.google.com/spreadsheets/d/1bWi6smdTMClreG96sGjYxb2lee9Ja6eT4-Lm7eXm1gM/edit#gid=976949241

**Devices:** Redmi Note 8 (10.0)

**Comment by MuckT** [comment]
https://valora.testquality.com requirement [R100](https://valora.testquality.com/project/12128/testcase/299604/requirement/11871).

**Comment by ValoraQA** [comment]
Hey @MuckT We have verified this issue using latest Test flight build V 1.19.0 and Android internal build V1.19.0 & updated status in Test Quality Tool

**Test Quality Tool link:**

**iOS (13.0):** 

- https://valora.testquality.com/project/12128/plan/21816/run/150277/result/2485155

**Verified On Devices:** iPhone SE (13.6.1)

**Pull request merged**